src/ooxml/java/org/apache/poi/xwpf/usermodel/XWPFChartData.java

```diff
@@ -0,0 +1,82 @@
+/* ====================================================================
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+==================================================================== */
+
+package org.apache.poi.xwpf.usermodel;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.apache.poi.POIXMLDocumentPart;
+import org.apache.poi.POIXMLException;
+import org.apache.poi.openxml4j.opc.PackagePart;
+
+/**
+ * Raw picture data, normally attached to a WordprocessingML Drawing.
+ * As a rule, embedded xlsx are stored in the /word/embedded/ part of a WordprocessingML package.
+ */
+public class XWPFChartData extends POIXMLDocumentPart {
+    
+    /**
+     * Create a new XWPFChartData node
+     * @since POI 4.0.0
+     */
+    protected XWPFChartData() {
```

Comment:
From the class name, I would have expected this class to be somewhat related to the `XDDFChartData` class. 

It also seems rather empty from behavior and only offers raw access to its input stream. Not sure what was your intent with that...

Reply:
This class we are using for embedded part of xwpfchart object. Till now we are reading chart embedded part from template file and then updating its value according to use.
with above mention class we can write XSSFWorkbook to embedded part of chart directly